frontend/apps/reader/modules/readerview.lua

```diff
@@ -73,6 +73,39 @@ local ReaderView = OverlapGroup:extend{
     currently_scrolling = false,
 }
 
+--- Returns color struct from a hex color string
+function parseColor(value)
```

Comment:
local

Reply:
May be it should be part of blitbuffer.lua (as it returns BlitBuffer structs), if this ends up being (ab)used by other colorful people.

> Since the color is constant within a page, 

I expect this to not last, people will want different color per highlight.

Reply:
> people will want different color per highlight

So it should go to `drawPageSavedHighlight` and `drawXPointerSavedHighlight` and be passed to `drawHighlightRect` as arg.

Reply:
@smasher816, as you are not going to make it for each highlight, something similar can be used so far:
https://github.com/koreader/koreader/blob/2d3356358761e565cfc1cf9a6cc6cfc81a95c121/frontend/apps/reader/modules/readerview.lua#L587